app/src/main/java/com/android/sample/model/recipe/RecipeRepository.kt

```diff
@@ -0,0 +1,6 @@
+package com.android.sample.model.recipe
+
+interface RecipeRepository {
```

Comment:
By looking at the SonarCloud, I saw that there is a suggestion on how to define this interface with only only one function. I let you take a look at it and maybe change it.

Reply:
Regarding the suggestion to make the interface functional, I opted not to turn it into a functional interface because we anticipate adding more functions in the future. This will provide flexibility for expanding the functionality of the repository.

Reply:
In that case, it's perfect. Thank you for the clarification.